### ISSUE TITLE
Extract data of ellipsoids

### ISSUE THREAD

**Issue by 3406212002** [graphdeco-inria/gaussian-splatting#996]
I want to get the mean and covariance of all ellipsoids of the fitting structure, is this achievable?

**Comment by AsherJingkongChen** [comment]
Yes, you can bind the CUDA kernel function such as `computeCov3d`. It is an easy task for AI coding tools.

**Comment by 3406212002** [comment]
> Yes, you can bind the CUDA kernel function such as `computeCov3d`, `transformXXX`

Hi,thanks for your kind reply,I will try that later.
